docs/reference.md

```diff
@@ -1873,9 +1873,9 @@ Key | Type | Required |
 [body](#body) | Object/String | No |
 [headers](#headers) | String | No |
 
-_nb: For multipart/form-data content type requests, the body should be provided as a key/value object. For other content types, it can be provided as String in JSON format._
+_nb: For multipart/form-data content type requests, the body should be provided as a key/value object. For other content types, String in JSON format is also supported._
 
-_nb: For a JSON api, the response can be processed in subsequent activities in SWADL using the utility function [json()](#utility-functions)_
+_nb: If the api response is not JSON, then it can be processed in subsequent activities in SWADL using the utility function [json()](#utility-functions)_
```

Comment:
if it not JSON content while would you use the json() function?

Reply:
I'm removing json() method in next commit. it is not needed anymore